--- cmake/FindDeflate.cmake
+++ cmake/FindDeflate.cmake
@@ -37,21 +37,27 @@
 set(Deflate_NAMES deflate deflatestatic)
 set(Deflate_NAMES_DEBUG deflated deflatestaticd)
 
+find_package(libdeflate CONFIG)
+if (TARGET libdeflate::libdeflate_static)
 find_path(Deflate_INCLUDE_DIR
           NAMES libdeflate.h
+          PATHS "${libdeflate_DIR}/../../.."
           PATH_SUFFIXES include)
+endif()
 
 set(Deflate_OLD_FIND_LIBRARY_PREFIXES "${CMAKE_FIND_LIBRARY_PREFIXES}")
 # Library has a "lib" prefix even on Windows.
 set(CMAKE_FIND_LIBRARY_PREFIXES "lib" "")
 
 # Allow Deflate_LIBRARY to be set manually, as the location of the deflate library
-if(NOT Deflate_LIBRARY)
+if(TARGET libdeflate::libdeflate_static AND NOT Deflate_LIBRARY)
   find_library(Deflate_LIBRARY_RELEASE
                NAMES ${Deflate_NAMES}
+               PATHS "${libdeflate_DIR}/../../.."
                PATH_SUFFIXES lib)
   find_library(Deflate_LIBRARY_DEBUG
                NAMES ${Deflate_NAMES_DEBUG}
+               PATHS "${libdeflate_DIR}/../../.."
                PATH_SUFFIXES lib)
 
   include(SelectLibraryConfigurations)
